.travis.yml

```diff
@@ -24,7 +24,7 @@ addons:
 matrix:
   include:
     # Doc build second as it also takes a long time to run
-    - env: Linux64Docbuild
```

Comment:
I guess the usage of `env` here was a typo.
If I am wrong then please let me know and I will revert this change!